### PR TITLE
Add custom field

### PR DESCRIPTION
**Pull request by Siilwyn**
- [x] commit messages are commitizen-friendly
- [x] code is unit tested
- [x] code is integration tested
- [x] public code is documented
- [x] code is reviewed by at least one person
- [x] code satisfies linter rules

### CODE REVIEW

**Comment by butenkor** [comment]
Same support for category sync is also planned?

File: src/coffee/sync/inventory-sync.coffee [hunk 1]
```diff
@@ -20,7 +21,7 @@ InventoryUtils = require './utils/inventory'
 #     # do nothing
 class InventorySync extends BaseSync
 
-  @actionGroups = ['quantity', 'expectedDelivery']
+  @actionGroups = ['quantity', 'expectedDelivery', 'custom']
```
Comment (hisabimbola) [inline src/coffee/sync/inventory-sync.coffee]: I remembered you briefly explained what this does to me. Can you add it as a comment as I cannot remember again 😞

Comment (Siilwyn) [inline src/coffee/sync/inventory-sync.coffee]: It is defined [on line 22](https://github.com/sphereio/sphere-node-sdk/blob/master/src/coffee/sync/base-sync.coffee#L22) of the base-sync file.

File: src/spec/integration/inventories-sync.spec.coffee [hunk 2]
```diff
@@ -101,3 +101,65 @@ describe 'Integration Inventories Sync', ->
       expect(result.body.expectedDelivery).not.toBeDefined()
       done()
     .catch (error) -> done(_.prettify(error))
+
+  describe 'custom type and field handling', (done) ->
+    customType = undefined
+    inventoryEntry = undefined
+
+    beforeEach (done) ->
+      @client = new SphereClient config: Config
```
Comment (Siilwyn) [inline src/spec/integration/inventories-sync.spec.coffee]: It has been done. 🎭

**Comment by hisabimbola** [comment]
@Siilwyn Tests are failing though, so maybe you check that.

**Comment by butenkor** [comment]
@Siilwyn custom fields are not supported for category-sync. Should this be part of this PR too?

**Comment by Siilwyn** [comment]
@butenkor no I'd like to limit the scope of this PR, sorry about the confusing branch naming. The inventory has priority for now, in the future all (category, order, product) should support custom fields.

**Comment by Siilwyn** [comment]
PR ready, @PhilippSpo or @emmenko merge please. :)

**Pull request merged**